app/styles/partials/utils.scss

```diff
@@ -18,6 +18,18 @@
   display: none !important;
 }
 
+.h-auto {
+  height: auto !important;
+}
+
+.h-100 {
+  height: 100% !important;
+}
+
+.h-80perc {
+  height: 80% !important;
+}
+
```

Comment:
why 80?

Reply:
Initially
![image](https://user-images.githubusercontent.com/56474333/118429811-25f2af80-b6f0-11eb-9946-f3363ba6a557.png)

Now
![image](https://user-images.githubusercontent.com/56474333/118429523-8503f480-b6ef-11eb-8b1d-15ba3b6a0481.png)

80% height seemed to give the desired output